app/views/casework/crime_applications/sections/_outgoing_payments.html.erb

```diff
@@ -0,0 +1,37 @@
+<% if outgoing_payments %>
+  <h2 class="govuk-heading-m">
+    <%= label_text(:payments_the_client_pays) %>
+  </h2>
+
+  <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+    <% if outgoing_payments.any? %>
+      <% outgoing_payments.each do |payment_type, value| %>
+        <%= render partial: 'payment_with_frequency',
+                   locals: {
+                     key: label_text("outgoing_payment_#{payment_type}"),
+                     payment: value,
+                     does_not_text: t('values.does_not_pay')
+                   } %>
+        <% if payment_type == 'legal_aid_contribution' && value.present? %>
```

Comment:
Nil payments are also included in the array so also need to check if the value is a payment or nil

Reply:
@hiboabd You might be rendering a partial with nil payment. What do you think of moving payment value check before render?